docs/man/dunes.1.md

```diff
@@ -202,6 +202,10 @@ contracts on Antelope blockchains.
 
     Display the current versions of DUNES, CDT, and leap.
 
+**`--version-short`**
+
+    Display the current version of DUNES in short format.
+
```

Comment:
Right now, `--version-short` is a hidden switch. I don't think it matters that it's documented here and not with `--help`, but maybe we should update it in dunes so it is displayed with help?

Reply:
> Right now, `--version-short` is a hidden switch. I don't think it matters that it's documented here and not with `--help`, but maybe we should update it in dunes so it is displayed with help?

So may be add info that it is hidden?